spec/models/section_spec.rb

```diff
@@ -165,14 +165,14 @@ def key_classes_for(hash)
   describe "#eql?" do
     let(:editions) { [draft_edition_v1] }
 
-    it "is considered the same as another section instance if they have the same id" do
+    it "is considered the same as another section instance if they have the same uuid" do
```

Comment:
I think this is asserting that a `Section` behaves in an ActiveRecord-like way, i.e. that two in-memory instances of objects representing the same record in a database will be "equal". Can you think of better wording to make this clearer?